games-roguelike/hengband/hengband-1.6.2-r1.ebuild

```diff
@@ -1,62 +1,63 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
-inherit eutils autotools games
+EAPI=7
+inherit autotools eutils
```

Comment:
is `eutils` even required here?

Reply:
Always audit the list of eclasses for unused stuff and sort them.